frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -88,7 +56,7 @@ const selectedUserTemplateUpdater = (options: UpdateOptions) => {
       [VMSettingsField.IMAGE_URL]: { isDisabled },
       [VMSettingsField.OPERATING_SYSTEM]: { isDisabled },
```

Comment:
I suppose we should disable also workload profile since it cannot be changed for the user template